### ISSUE TITLE
npm index.js is outdate and has no free api

### ISSUE THREAD

**Issue by jumoog** [funkyremi/deepl#9]
you forgot todo tsc?

**Comment by funkyremi** [comment]
No.
Can you give more informations and provide an example to reproduce your error?

**Comment by jumoog** [comment]
`npm install deepl`

```JSON
{
  "dependencies": {
    "deepl": "^1.0.8"
  }
}
```

`node_modules\deepl\dist\index.js`

has no `free_api` Parameter 

``` Javascript
const translate = require("deepl");

translate({
    free_api: true,
    text: 'I am a text',
    target_lang: 'FR',
    auth_key: 'FREE-API-KEY',
    // All optional parameters available in the official documentation can be defined here as well.
  })
  .then(result => {
      console.log(result.data);
  })
  .catch(error => {
      console.error(error)
  });
```

**HTTP Response:**

```JSON
"data": { "message": "Wrong endpoint. Use https://api-free.deepl.com" }
```

**Comment by FunctionDJ** [comment]
Same here

**Comment by funkyremi** [comment]
Ok indeed I forgot the build. Sorry for the inconvenience.
Just published a new version 1.0.9.
